src/main/java/DayThree.java

```diff
@@ -14,6 +16,7 @@
         else{
             if(resultList.get(numberToCheck).equals("1"))
             {
+                //valueOf vs parse plus duplication
```

Comment:
As before I think `Integer.parseInt` would be more appropriate.  Also there's a lot of duplication here for add and multiple, can we reduce it?